gnosis/gp-ui
# [Explorer] Add a tooltip arrow below the icon

# Summary

Closes #722 

Added a tooltip arrow on the icon instead of the text itself.


<img width="375" alt="Screen Shot 2021-10-20 at 12 30 44" src="https://user-images.githubusercontent.com/11525018/138123971-d6eacb68-4c20-43e5-8fa6-34b5effa909a.png">

# To Test

1. Open the page `/rinkeby/address/<any address with orders>/` (i.e [0xb6bad41ae76a11d10f7b0e664c5007b908bc77c9](https://pr721--gpui.review.gnosisdev.com/rinkeby/address/0xb6BAd41ae76A11D10f7b0E664C5007b908bC77C9/))
* View it in mobile view
* Click on the clock icon. Tooltip should appear with the arrow pointing to the icon.
